source/menus/quest_status_menu.cpp

```diff
@@ -264,6 +313,29 @@ void QuestStatusMenu::draw() {
         dComIfGs_setShield(new_shield_item_id);
         dComIfGs_setShieldOwned(shieldIdToShieldOwned(new_shield_item_id));
         break;
+    case MENU_ITEM_MAGIC:
+        new_max_magic_value = dComIfGs_getMaxMagic();
+        Cursor::moveListSimple(new_max_magic_value);
+        if (new_max_magic_value == NO_MAGIC - 1) {
+            new_max_magic_value = NO_MAGIC;
+        } else if (new_max_magic_value == NO_MAGIC) {
+            new_max_magic_value = NO_MAGIC;

```

Comment:
This "else if" is redundant:  if new_max_magic_value is equal to NO_MAGIC, then set new_max_magic_value equal to NO_MAGIC. 

new_max_magic_value is already equal to NO_MAGIC